Mathlib/Analysis/Convex/Schur.lean

```diff
@@ -0,0 +1,153 @@
+/-
+Copyright (c) 2025 Bhavik Mehta. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Bhavik Mehta
+-/
+
+import Mathlib.Analysis.SpecialFunctions.Pow.Real
+import Mathlib.Order.Monotone.Monovary
+
+/-!
+# Schur's inequality
+
+This file proves a number of forms of Schur's inequality.
+The proofs are adapted from https://www.cip.ifi.lmu.de/~grinberg/VornicuS.pdf
+-/
+
+lemma vornicu_schur_mildorf {α : Type*} [PartialOrder α] [CommRing α] [IsOrderedRing α]
```

Comment:
As a general comment, I'm not schur whether we should diverge from the naming convention here.

Reply:
For named inequalities, my preference is to have both names, one as an alias, for readability and discoverability. What do you think?

Reply:
I think if you want that we should modify the naming guide to explicitly allow it (via a Zulip discussion)

Reply:
Sure, but note that this pattern already exists in mathlib (see `law_cos`) and was discussed here: https://leanprover.zulipchat.com/#narrow/channel/345428-mathlib-reviewers/topic/Symbol-reading.20as.20a.20naming.20convention.

Reply:
Let's encode this into #naming then and ping the thread on the doc PR?